src/main/markdown/gettingstarted.md

```diff
@@ -162,3 +163,18 @@ set up Eclipse to use the GWT SDK:
 
 If you are going to stick with the command line, check out Speed Tracer     and then
 head over to [Build a Sample GWT App](doc/latest/tutorial/gettingstarted.html).
+
+## Maven<a id="maven"></a>
+
+If you're aiming at building your app with Maven you can use [gwt-maven-plugin](http://mojo.codehaus.org/gwt-maven-plugin/). See the plugin documentation for more information.
+
+To get you started, here's how to generate a new project based on the archetype.
+
+```
+mvn archetype:generate \
```

Comment:
While I fixed most warts in this archetype, I still wouldn't recommend it except for one-off experiments or bug repro cases.